shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -357,9 +357,10 @@ - (void)testUpdateViewportMetricsDoesInvokeEngineWhenIsTheViewController {
                                                                                 nibName:nil
                                                                                  bundle:nil];
   mockEngine.viewController = viewController;
-  [viewController updateViewportMetrics];
   flutter::ViewportMetrics viewportMetrics;
-  OCMVerify([mockEngine updateViewportMetrics:viewportMetrics]);
+  OCMExpect([mockEngine updateViewportMetrics:viewportMetrics]).ignoringNonObjectArgs();
```

Comment:
The viewport metrics aren't the same on different simulators. But the point of this is that `updateViewportMetrics` is called, so ignore the actual viewport metric values.